front/src/services/study/api.ts

```diff
@@ -76,8 +76,8 @@ export const fetchAllWorkFlows= () => {
 export const fetchReactionsIsland= (nctId: any) => {
     return callGraphql(ENDPOINT, query.REACTIONS_ISLAND_QUERY, {nctId});
 };
-export const deleteReaction = (id:any) =>{
-    return callGraphql(ENDPOINT,mutate.DELETE_REACTION, {id})
+export const deleteReaction = (id:any, studyQuery: any, nctId: any) =>{
+    return callGraphql(ENDPOINT,mutate.DELETE_REACTION, {id, studyQuery, nctId})
```

Comment:
I don't believe we need to pass these other params(studyQuery and nctId) besides just the id to be deleted, is it working w no errors?